packs/data/equipment-effects.db/effect-nosoi-charm-greater-lifesense.json

```diff
@@ -1,45 +0,0 @@
-{
-    "_id": "0rkg7SVzs2AetUXw",
-    "img": "systems/pf2e/icons/equipment/held-items/nosoi-charm.webp",
-    "name": "Effect: Nosoi Charm (Greater) (Lifesense)",
-    "system": {
-        "badge": null,
-        "description": {
-            "value": "<p>You gain lifesense as an imprecise sense with a range of 30 feet as long as you are holding the charm, and do not need to activate it. This allows you to sense the life force that animates living creatures and the perverse force that animates the dead, though you can't distinguish between the two.</p>"
```

Comment:
I assume these REs have been moved to the item itself?

Reply:
They have been, yes